### ISSUE TITLE
Bronchoconstriction

### ISSUE THREAD

**Issue by dede2021** [obophenotype/human-phenotype-ontology#6838]
**Preferred term label:**
Bronchoconstriction

**Synonyms**
- Bronchial Constriction
- Constriction of the bronchi

**Definition (free text, please give PubMed ID)**
- Tightening of smooth muscle surrounding the bronchi and bronchioles with consequent wheezing and shortness of breath ([PMID:26694039](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4940044/)).

- Narrowing of the caliber of the BRONCHI, physiologically or as a result of pharmacological intervention ([MESH:D016084](https://meshb.nlm.nih.gov/record/ui?ui=D016084)).

- Narrowing of air passage lumina, typically due to bronchial smooth muscle contraction, and leads to decreased air flow ([EFO:0009836](http://www.ebi.ac.uk/efo/EFO_0009836)).

**Parent term (use hpo.jax.org/app)**
Abnormal bronchus morphology (HP:0025426)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
- OMIM #208550
- OMIM #600807
- OMIM #130700

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added, but as child of Abnormal respiratory system physiology
